src/VisualizationTests/WpfVisualizationTests.csproj

```diff
@@ -188,6 +188,9 @@
   <ItemGroup>
     <WCFMetadata Include="Service References\" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Visual Studio adds them every time you run the tests.
@ikeough added them to a couple projects recently, but apparently there are still more.